### PR TITLE
Declare HTML 5 doctype for console and reduce logspam

### PR DESCRIPTION
**Pull request by SubJunk**
This addresses #2912

### CODE REVIEW

File: src/main/java/net/pms/network/mediaserver/javahttpserver/RequestHandler.java [hunk 1]
```diff
@@ -1369,11 +1370,15 @@ private static boolean filterIp(InetAddress inetAddress) {
 
 	private static void logMessageSent(HttpExchange exchange, String response, InputStream iStream, RendererConfiguration renderer) {
 		StringBuilder header = new StringBuilder();
+		boolean isXml = false;
 		for (Map.Entry<String, List<String>> headers : exchange.getResponseHeaders().entrySet()) {
 			String name = headers.getKey();
 			if (StringUtils.isNotBlank(name)) {
 				for (String value : headers.getValue()) {
 					header.append("  ").append(name).append(": ").append(value).append("\n");
+					if ("content-type".equalsIgnoreCase(name) && "/XML".contains(value.toUpperCase())) {
```
Comment (esabol) [inline src/main/java/net/pms/network/mediaserver/javahttpserver/RequestHandler.java:1379]: I think `"/XML".contains(value.toUpperCase())` is backwards. It should be `value.toUpperCase().contains("/XML")`, no?

Comment (SubJunk) [reply src/main/java/net/pms/network/mediaserver/javahttpserver/RequestHandler.java:1379]: Good point!

**Review by esabol — APPROVED** [review]

**Comment by SurfaceS** [comment]
V11 log all the traffic and already take care of parsing only XML.
I think this will mess it.
All of that for the "console" webpage that no one see the goal.

**Comment by SubJunk** [comment]
thanks for the investigation, this can be closed since it's already fixed in v11. i'll release the first v11 alpha version soon, there are just 2 bugs to fix first

**Pull request closed without merging**